src/js/calc.js

```diff
@@ -0,0 +1,47 @@
+export class Calc {
+  constructor() {
+    this.x = ''
+    this.y = ''
+    this.operator = ''
+  }
+
+  #isSetOperator() {
+    if (!this.x) return window.alert('처음엔 숫자를 입력해주세요.')
+
+    if (this.operator) return window.alert('연산자 다음엔 숫자를 입력해주세요.')
+  }
+
+  #isSetNumber() {
+    if (this.x.length < 3 && !this.operator) return true
+
+    if (this.y.length < 3 && this.operator) return true
+  }
+
+  setNum(number) {
+    if (!this.operator) this.#isSetNumber() ? (this.x += number) : window.alert('숫자는 세자리수까지 입력 가능합니다.')
+    else this.#isSetNumber() ? (this.y += number) : window.alert('숫자는 세자리수까지 입력 가능합니다.')
+  }
```

Comment:
```suggestion
  setNum(number) {
    if (!this.#isSetNumber()) {
      return window.alert('숫자는 세자리수까지 입력 가능합니다.');
    }
    if (!this.operator) {
      this.x += number;
    } else {
      this.y += number;
    }
  }
```
이렇게 표현할 수 있겠네요!

그런데 이보다 더 좋은 방법이 있지 않을까요?